book/src/2.getting-started/3.running-in-teams.md

```diff
@@ -1,54 +1,88 @@
 # Running in Teams
 
-Now that you have your agent running locally, let's deploy it to Microsoft Teams for testing. This guide will walk you through the process of running your agent directly in Teams.
+Now that your agent is running locally, let's deploy it to Microsoft Teams for testing. This guide will walk you through the process.
 
-## Installing Teams Toolkit
+## Teams Toolkit
 
-First, you'll need to install the Teams Toolkit extension:
+Teams Toolkit is a powerful tool that simplifies deploying and debugging Teams applications. It automates tasks like managing the Teams app manifest, configuring authentication, provisioning, and deployment.
 
-1. Visit the [Teams Toolkit installation guide](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode)
-2. Follow the instructions to install for your preferred IDE
+### Install Teams Toolkit (TTK)
 
-> [!TIP]
-> The Teams Toolkit extension makes it easy to debug and test your Teams applications directly in Microsoft Teams. If you'd like to learn the various things that it helps automate, check out [this doc](../4.teams/1.core-concepts.md)
+First, you'll need to install the Teams Toolkit IDE extension:
 
-## Adding Teams Configuration
+1. Visit the [Teams Toolkit installation guide](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode).
+2. Follow the instructions to install for your preferred IDE.
 
-Your agent needs some basic configuration to work with Teams. Add this by running:
+> [!NOTE]<br/>
```

Comment:
Kinda a repetition of the above

Reply:
this was pre-existing, so I left it. Do you want me to delete it?

Reply:
nvm I combined it with line 7